includes/REST_API/Stories_Controller.php

```diff
@@ -115,6 +117,10 @@ public function prepare_item_for_response( $post, $request ) {
 			$data['preview_link'] = $view_link;
 		}
 
+		if ( in_array( 'embed_post_link', $fields, true ) && current_user_can( 'edit_posts' ) ) {
+			$data['embed_post_link'] = add_query_arg( [ 'from-web-story' => $post->ID ], admin_url( 'post-new.php' ) );
+		}
```

Comment:
@swissspidy I made `embed_post_link` optional, if the user can not edit posts. Makes no sense to show this link then take then to a permission error page.

Reply:
<img width="1904" alt="Screenshot 2021-07-13 at 11 38 41" src="https://user-images.githubusercontent.com/237508/125438268-390d3440-af83-4b8a-b456-a2681a76b784.png">

This is what it looks like without a `embed_post_link`

Reply:
Awesome, thank you!